DiamondLightSource/dodal
# 76 make stage synchronous if arming not started

Fixes #76 

I think this should wait on #43 before being reviewed

To test:
Run new tests and check they are sensible
Check changes satisfy the criteria 
See if this fixes https://github.com/DiamondLightSource/python-artemis/issues/686
